consensus/src/processes/pruning_proof/mod.rs

```diff
@@ -258,18 +331,18 @@ impl PruningProofManager {
                             mergeset_reds: calculated_gd.mergeset_reds.clone(),
                             blues_anticone_sizes: calculated_gd.blues_anticone_sizes.clone(),
```

Comment:
```suggestion
                            mergeset_reds: calculated_gd.mergeset_reds,
                            blues_anticone_sizes: calculated_gd.blues_anticone_sizes,
```

same goes for the line above `mergeset_blues: calculated_gd.mergeset_blues,`